Explorer/Assets/DCL/MarketplaceCredits/MarketplaceCreditsMenuController.cs

```diff
@@ -235,15 +235,19 @@ public override void Dispose()
             sidebarButtonStateCts.SafeCancelAndDispose();
 
             marketplaceCreditsAPIClient.OnProgramProgressUpdated -= SetSidebarButtonState;
-            viewInstance!.OnAnyPlaceClick -= OnAnyPlaceClicked;
-            viewInstance.InfoLinkButton.onClick.RemoveListener(OpenInfoLink);
-            viewInstance.TotalCreditsWidget.GoShoppingButton.onClick.RemoveListener(OpenGoShoppingLink);
-
-            marketplaceCreditsWelcomeSubController.Dispose();
-            marketplaceCreditsVerifyEmailSubController.Dispose();
-            marketplaceCreditsGoalsOfTheWeekSubController.Dispose();
-            marketplaceCreditsWeekGoalsCompletedSubController.Dispose();
-            marketplaceCreditsProgramEndedSubController.Dispose();
+
+            if (viewInstance != null)
+            {
+                viewInstance.OnAnyPlaceClick -= OnAnyPlaceClicked;
+                viewInstance.InfoLinkButton.onClick.RemoveListener(OpenInfoLink);
+                viewInstance.TotalCreditsWidget.GoShoppingButton.onClick.RemoveListener(OpenGoShoppingLink);
+            }
+
+            marketplaceCreditsWelcomeSubController?.Dispose();
```

Comment:
All these fields should be defined as nullable `?`

Reply:
Done! https://github.com/decentraland/unity-explorer/pull/4048/commits/94fadd8a8efa85f6f8543f3e75a0fc874d3c877b